package.json

```diff
@@ -17,7 +17,7 @@
     "demo8": "node dist/js/demo/8_demo_verifier.js",
     "debug": "node --inspect-brk dist/js/index.js",
     "prebuild": "rm -rf dist/js",
-    "build": "yarn compile && yarn build:typechain && tsc",
+    "build": "npm run compile && npm run build:typechain && tsc",
```

Comment:
I don't have much of an opinion, but why the change from yarn to npm?

Reply:
in the repo, someone was using both, so I've decided to go with npm as it is already included in node :)
that's the only reason :)